Tests/XCUITests/PhotonActionSheetTests.swift

```diff
@@ -93,6 +96,8 @@ class PhotonActionSheetTests: BaseTestCase {
     // https://testrail.stage.mozaws.net/index.php?/cases/view/2323203
     func testShareSheetSendToDevice() {
         openNewShareSheet()
+        mozWaitForElementToExist(app.staticTexts["Open in Firefox"])
+        mozWaitForElementToExist(app.staticTexts["Send to Device"])
```

Comment:
Sometimes, "Send to Device" is not ready before `tap()`. 😞

Reply:
Same here, just wait for the element you want to tap if the other is going to appear at the same time, same view